Packs/Jira/Integrations/JiraV3/JiraV3.yml

```diff
@@ -0,0 +1,2199 @@
+      description: The key of the outward issue.
+      isArray: false
+      name: outward_issue
+      required: true
+      secret: false
+    - default: false
+      description: The key of the inward issue.
+      isArray: false
+      name: inward_issue
+      required: true
+      secret: false
+    - default: false
+      description: The name of the link to apply. Use `jira-issue-link-type-get` to see available links.
+      isArray: false
+      name: link_type
+      required: true
+      secret: false
+    - default: false
+      description: A comment to add to the inward issue.
+      isArray: false
+      name: comment
+      required: false
+      secret: false
+    deprecated: false
+    description: Use this command to create a link between two issues.
+    execution: false
+    name: jira-issue-to-issue-link
+  - deprecated: false
+    description: Dummy fetch
+    execution: false
+    name: fetch-incidents
+  - arguments:
+    - default: false
+      description: The remote incident ID.
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    - default: false
+      defaultValue: '0'
+      description: The UTC timestamp in seconds of the last update. The incident is only updated if it was modified after the last update time.
+      isArray: false
+      name: lastUpdate
+      required: false
+      secret: false
+    deprecated: false
+    description: Get remote data from a remote incident. This method does not update
+      the current incident, and should be used for debugging purposes only.
+    execution: false
+    name: get-remote-data
+  - deprecated: false
+    description: Returns the list of fields to map in outgoing mirroring.
+      This command is only used for debugging purposes.
+    execution: false
+    name: get-mapping-fields
+  - arguments:
+    - default: false
+      description: Date string representing the local time. The incident is only returned if it was modified after the last update time.
+      isArray: false
+      name: lastUpdate
+      required: false
+      secret: false
+    deprecated: false
+    description: Get the list of incidents that were modified since the last update time.
+      This method is used for debugging purposes. The get-modified-remote-data command
+      is used as part of the Mirroring feature that was introduced in Cortex XSOAR version 6.1.
+    execution: false
+    name: get-modified-remote-data
+  - deprecated: false
+    description: Updates the remote incident with local incident changes. This method
+      is only used for debugging purposes and will not update the current incident.
+    execution: false
+    name: update-remote-system
+  dockerimage: demisto/btfl-soup:1.0.1.47958
+  feed: false
+  isfetch: true
+  ismappable: true
+  isremotesyncin: true
+  isremotesyncout: true
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
+  - no tests
```

Comment:
I see you have a TPB, please add it here :) 
Also, don't forget to add the right definition in conf.json